federation/pkg/federation-controller/ingress/ingress_controller.go

```diff
@@ -158,6 +215,25 @@ func NewIngressController(client federation_release_1_4.Interface) *IngressContr
 			err := client.Extensions().Ingresses(ingress.Namespace).Delete(ingress.Name, &api.DeleteOptions{})
 			return err
 		})
+
+	// Federated configmap updater along with Create/Update/Delete operations.  Only Update should ever be called.
+	ic.federatedConfigMapUpdater = util.NewFederatedUpdater(ic.configMapFederatedInformer,
+		func(client kube_release_1_4.Interface, obj pkg_runtime.Object) error {
+			configMap := obj.(*api.ConfigMap)
+			configMapName := types.NamespacedName{Name: configMap.Name, Namespace: configMap.Namespace}
+			return fmt.Errorf("Internal error: Incorrectly attempting to create ConfigMap: %q", configMapName)
+		},
+		func(client kube_release_1_4.Interface, obj pkg_runtime.Object) error {
+			configMap := obj.(*v1.ConfigMap)
+			glog.V(4).Infof("Attempting to update ConfigMap: %v", configMap)
```

Comment:
Yes, agreed.  Will do, and add appropriate release notes.  It would actually be pretty easy to refuse to do this here if any Ingresses already exist in the cluster.  See also https://github.com/kubernetes/contrib/issues/1656 for the better plan for beta and GA, as this is clearly a fairly significant limitation of Federated Ingress alpha.